release-team/role-handbooks/ci-signal/README.md

```diff
@@ -98,23 +99,23 @@ Increased attention on maintaining signal early in the release cycle goes a long
 ### Burndown to Code Freeze
 
 This is when things really begin to ramp up in the release cycle with active bug triaging and followup on possible release blocking issues to assess release health. Day to day tasks of CI Signal lead include
-- Auditing test status of master-blocking, master-upgrade and release-1.x.y-blocking dashboards on **daily basis**
+- Auditing test status of master-blocking, master-informing, release-1.x.y-blocking, and release-1.x.y-informing dashboards on **daily basis**
 - Keeping issues' status upto-date on GitHub
 - Working closely with SIGs, test owners, bug triage lead and Release team in triaging, punting and closing issues
 - Updating issue tracker frequently so anyone on the release team can get to speed and help followup on open issues
 - Continuing to send [weekly CI signal report](#reporting-status)
 
 ### During Code Freeze
 
-- Continue best practices from Burndown stage. Monitor master-blocking, master-upgrade and release-1.x.y-blocking dashboards on **daily basis**
+- Continue best practices from Burndown stage. Monitor master-blocking, master-informing, release-1.x.y-blocking, and release-1.x.y-informing dashboards on **daily basis**
 - Quickly escalate any new failures to release team and SIGs
 - Continuing to send [weekly CI signal report](#reporting-status)
 
 ### Exit Code Freeze
 
 - Once 1.x.y-rc.1 release branch is cut and master re-opens for next release PRs
-  - continue **monitoring master-upgrade and 1.x.y-blocking dashboards on daily basis**
-  - you need not monitor master-blocking on a daily basis. It is, however, worth keeping an eye on master-blocking especially before risky cherry-picks make their way into the release branch
+  - continue **release-1.x.y-blocking and release-1.x.y-informing dashboards on daily basis**
+  - you need not monitor master-blocking or master-informing on a daily basis. It is, however, worth keeping an eye on master-blocking especially before risky cherry-picks make their way into the release branch
 - If tests have stabilized (they should have by now), you may stop sending the weekly CI report
```

Comment:
you actually need to monitor master-informing post-code-freeze, because of the performance tests.  I have a separate PR open for that text, though.

Reply:
aah thank you for catching this one!